src/projectscene/qml/Audacity/ProjectScene/trackspanel/RealtimeEffectListItem.qml

```diff
@@ -0,0 +1,134 @@
+import QtQuick
+import QtQuick.Controls
+import QtQuick.Layouts
+
+import Muse.Ui
+import Muse.UiComponents
+
+ListItemBlank {
+    id: root
+
+    property var item: null
+    property var availableEffects: null
+    property var handleMenuItemWithState: null
+
+    height: 24
+    clip: false // should be true?
+    background.color: "transparent"
```

Comment:
you can use `normalColor`

Reply:
... which is the default, so I can remove this line. Thanks !